.codecov.yml

```diff
@@ -5,3 +5,8 @@ codecov:
   # DAutoTester and auto-tester are not in the default list
   ci:
     - !travis
+
+coverage:
+  fixes:
+    - "test/.*/::src/"
+    - "fail_compilation/::src/"
```

Comment:
I just referred to the second line, but I guess no one minds if it's necessary for your side ;-)